etc/kayobe/inventory/group_vars/all/firewall

```diff
@@ -0,0 +1,323 @@
+---
+#########################################
+# StackHPC default firewall configuration
+#########################################
+# This file contains the reference StackHPC firewalld rule implementation. It
+# is designed to cover as many services and use cases as possible, however may
+# not be compatible with all deployments. For more information, see the
+# firewall page in the configuration section of the SKC docs.
+
+# A templated list of firewalld zones, according to the host's network
+# interfaces.
+# This variable can be used to set any *_firewalld_zones kayobe variable e.g.
+# compute_firewalld_zones: "{{ stackhpc_firewalld_zones }}" in compute.yml
+stackhpc_firewalld_zones: |
+  {% set network_zones = [] %}
+  {% for network in network_interfaces %}
+  {% if network | net_zone %}
+  {% set _ = network_zones.append({'zone': network | net_zone }) %}
+  {% endif %}
+  {% endfor %}
+  {{ network_zones | unique | list }}
```

Comment:
This final `| list` probably doesn't need to be here.

Reply:
Filters like unique tend to return a Python generator rather than a proper list, and they don't always act as you would expect.

Reply:
Interesting, can leave as is then